File: 프로그래머스/0/120956. 옹알이 （1）/옹알이 （1）.java
class Solution {
  public static int solution(String[] babbling) {
    int answer = 0;
    
    String [] babblecase= new String [] {"aya", "ye", "woo", "ma"};
    
    for (int i = 0; i < babbling.length; i++) {
    	int count=0;
		String s=babbling[i];
		for (int j = 0; j < babblecase.length; j++) {
			count+=s.contains(babblecase[j])?babblecase[j].length():0;
		}
		answer+=s.length()==count?1:0;	}
	return answer;
}
}

File: 프로그래머스/0/120956. 옹알이 （1）/옹알이 （1）.js
function solution(babbling) {
  const babblingpossible = ["aya", "ye", "woo", "ma"];

  var answer = 0;
  babbling.forEach((element) => {
    let count = 0;
    babblingpossible.forEach((elementp) => {
      count += element.indexOf(elementp) !== -1 ? elementp.length : 0;
    });

    answer += element.length === count ? 1 : 0;
   
  });

  return answer;
}